pilot/pkg/proxy/envoy/v2/xds_test.go

```diff
@@ -31,6 +31,10 @@ import (
 	"encoding/json"
```

Comment:
can you just rename this to ads_test.go ?
and if there is not much left in all the other LDS/CDS/EDS, merge them all into one ads.go ?